stack-ghc861.yaml

```diff
@@ -0,0 +1,24 @@
+resolver: nightly-2018-10-15  # GHC 8.6.1
+
+packages:
+- haskell-indexer-backend-core
+- haskell-indexer-backend-ghc
+- haskell-indexer-frontend-kythe
+- haskell-indexer-pathutil
+- haskell-indexer-pipeline-ghckythe
+- haskell-indexer-pipeline-ghckythe-wrapper
+- haskell-indexer-translate
+- kythe-proto
+- kythe-schema
+- text-offset
+
+extra-deps:
+
```

Comment:
How come we don't need the extra deps for ghc861? Although, CI is passing for this version. I might be missing something...

Reply:
Because it's using the nightly resolver which has the right packages already.